docs/client_tools/companion/katana.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Katana Submission
+summary: Submit a Katana Renderman scene to Conductor for rendering.
+authors:
+    - Julian Mann
+date: 2020-01-29
+---
+
+
+# Katana Renderman example
+
+## Get prepared
+
+--8<-- "client_tools/snippets/install_pointer.md"
+
+In this tutorial, you'll set up a Katana scene that uses Pixar's PRman and then render it on Conductor. 
+
+We don't provide a downloadable demo scene for this tutorial, so go ahead and build a simple scene with some objects that reference some file textures. Your node graph should look similar to the image below.
+
+![katana_graph][katana_graph]
+
+### Windows paths
+
+When your scene and assets are shipped to Conductor the filesystem hierarchy is set to mirror your local filesystem. However the render nodes run Linux so there are no drive letters in the paths. For this reason, it's best to use parameter expressions to [make paths relative to the katana project file][multi_platform]. By using expressions your project file becomes portable across different platforms.
+
+Let's suppose you have the following project:
+
+```C:/Users/julia/projects/kat/ocean.katana```
+
+and it contains assets:
+
+```
+C:/Users/julia/projects/kat/assets/3d/shark.abc
+C:/Users/julia/projects/kat/assets/2d/shark.jpg
+C:/shared/assets/3d/water_surface.abc
+C:/shared/assets/2d/water_surface.jpg
+```
+
+and the render path in your **RenderOutputDefine** node is:
+
+```C:/Users/julia/projects/kat/renders/out.#.exr```
+
+To make these assets relative to the project, right-click on each filename parameter, choose 'Expression' as the Value Mode and enter the following expressions respectively.
+
+```
+project.dir + "/assets/3d/shark.abc"
+project.dir + "/assets/2d/shark.jpg"
+project.dir + "/./../../../shared/assets/3d/water_surface.abc"
+project.dir + "/../../../../shared/assets/2d/water_surface.jpg"
+```
+
+Similarly for the renderLocation:
+
+```
+project.dir + "/renders/out.#.exr"
+```
+
+You can see the fully resolved expression in the parameter, and your project should render locally just fine.
+
+![katana_expr][katana_expr]
+
+
+## Configure the submission
+
+* Open the Companion app, sign in to your account, and go to the **Submission Kit** page.
+
+#### General
+
+* Fill in the form fields in the **General** tab as illustrated below. You may have to choose a different instance type and a project that exists in your account.
+* Browse for the directory where your project renders to.
+
+
+![general][general]
+
+!!!note
+    Due to a limitation in the file dialog, the button to choose a directory is labeled "Upload" on some platforms. Don't worry, nothing is uploaded at this time.
```

Comment:
I would simplify this to just:
`The button to choose a directory may be labeled "Upload" on your computer. Don't worry, nothing gets uploaded at this time.`